charts/data-dashboard-backend/values.yaml

```diff
@@ -11,7 +11,6 @@ image:
   # -- image pull policy
   pullPolicy: Always
   # -- Overrides the image tag whose default is the chart appVersion.
-  tag: latest
```

Comment:
This shouldn't be removed since it is referenced in here
https://github.com/RADAR-base/radar-helm-charts/blob/main/charts/data-dashboard-backend/templates/deployment.yaml#L35

You can leave the value empty and it should default to the `.Chart.AppVersion`